extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -38,7 +41,7 @@ public function getContext( Title $title, $skinName ) {
 			$context = [
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$context". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)